raiden/blockchain/events.py

```diff
@@ -297,10 +297,6 @@ def poll_blockchain_events(self, block_number: BlockNumber) -> Iterable[DecodedE
                 yield decode_raiden_event_to_internal(event_listener.abi, self.chain_id, log_event)
 
     def uninstall_all_event_listeners(self) -> None:
-        for listener in self.event_listeners:
-            if listener.filter.filter_id:
-                listener.filter.web3.eth.uninstallFilter(listener.filter.filter_id)
```

Comment:
All filters used are "stateless", meaning we don't install the filter in the client. This was necessary because the clients don't persist the filters, and if the ethereum node is restarted the filter was simply gone.